helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -116,21 +118,35 @@ internal class ListLicensesCommand : CliktCommand(
 
         fun isPathExcluded(path: String) = ortResult.repository.config.excludes.findPathExcludes(path).isNotEmpty()
 
-        ortResult
+        val findingsByProvenance = ortResult
             .getLicenseFindingsById(packageId, applyLicenseFindingCurations)
-            .filter { (license, _) -> !onlyOffending || violatedRulesByLicense.contains(license) }
-            .mapValues { (license, locations) ->
-                locations.filter {
-                    !omitExcluded || !isPathExcluded(it.path) ||
-                            ignoreExcludedRuleIds.intersect(violatedRulesByLicense[license].orEmpty()).isNotEmpty()
+            .mapValues { (_, locationsByLicense) ->
+                locationsByLicense.filter { (license, _) ->
+                    !onlyOffending || violatedRulesByLicense.contains(license)
+                }.mapValues { (license, locations) ->
+                    locations.filter {
+                        !omitExcluded || !isPathExcluded(it.path) ||
+                                ignoreExcludedRuleIds.intersect(violatedRulesByLicense[license].orEmpty()).isNotEmpty()
+                    }
+                }.mapValues { (_, locations) ->
+                    locations.groupByText(sourcesDir)
                 }
             }
-            .mapValues { it.value.groupByText(sourcesDir) }
-            .writeValueAsString(
+
+        buildString {
+            appendln("  scan results:")
+            findingsByProvenance.keys.forEachIndexed { i, provenance ->
+                appendln("    [$i] ${provenance.writeValueAsString()}")
+            }
+        }.let { println(it) }
```

Comment:
Nit (even if similar code existed before): As the return type is not needed here, it would be a bit more common to use `also` instead of `let`.